packages/utils/package.json

```diff
@@ -45,7 +45,8 @@
     "js-yaml": "^3.13.1",
     "winston": "^3.3.3",
     "winston-daily-rotate-file": "^4.5.5",
-    "winston-transport": "^4.3.0"
+    "winston-transport": "^4.3.0",
+    "find-up": "^5.0.0"
```

Comment:
This module relies on being run on NodeJS but utils is used in the browser. Why should this utils be here and not in the CLI only?